src/common/components/ui/formik/constants.tsx

```diff
@@ -29,11 +29,10 @@ export const acmClusterNameValidationMessages = (t: TFunction) => ({
 });
 
 export const nameValidationMessages = (t: TFunction) => ({
-  INVALID_LENGTH: t('ai:1-253 characters'),
+  INVALID_LENGTH: t('ai:1-63 characters'),
```

Comment:
We'll need to check with CIM too because the `richNameValidationSchema` is being used in a few places, looks like not only for host names.

Reply:
How about keeping this function intact and introducing a specific one for a hostname `hostnameValidationMessages`? CIM uses this as a general validation for a resource name.

Reply:
We already had `hostnameValidationMessages` use by both OCM and CIM. I've renamed it to `BMCValidationMessages` (for you to use in CIM) and created new `hostnameValidationMessages` to go with the new `richHostnameValidationSchema` for our purposes in ocm/common.

These is likely a lot of duplicate code now, but 
1. CIM side should be unaffected
2. dealing with different validations based on different conditions in Yup schemas is a pain in the...